.pre-commit-config.yaml

```diff
@@ -39,9 +39,10 @@ repos:
 
   # Markdown formatting
   - repo: https://github.com/executablebooks/mdformat
-    rev: 0.7.17
+    rev: 1.0.0
```

Comment:
Updating the rev, otherwise pipelines fail from using an old version of package that is no longer supported.